source/main.js

```diff
@@ -1,29 +1,39 @@
~// Import modules
~
~import { onStart, onReset, checkState, updateState, timer, setCustomTime, keyboardShortcut, revealSettings, hideSettings} from './modules/timer.js';
~import { revealHelp, hideHelp } from './modules/help.js';
 import { showNotif, getNotificationStatus, playSound, getAlarm } from './modules/notifications.js';
 import { colorChange } from './modules/color-change.js';
 import { breakReminders } from './modules/breakReminder.js'
-import { addTaskButton, cancelTask, saveTask, createCustomTaskTag, finishTask } from './modules/task-list.js'
+import { addTaskButton, cancelTask, saveTask, createCustomTaskTag, clearAllTasks, clearCompletedTasks } from './modules/task-list.js'
 
 // Timer
 document.getElementById('form-enabler').addEventListener('change', setCustomTime);
 document.getElementById('start-button').addEventListener('click', onStart);
 document.getElementById('reset-button').addEventListener('click', onReset);
 
 // Keyboard shortcuts
-document.addEventListener('keydown', keyboardShortcut);
+document.addEventListener('keydown', (event) => {
+    // only allow this event to be fired when task form is hidden
+    if (document.getElementById("add-task-form").classList.contains("hidden"))
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/cse112-sp22-group5/cse112-sp22-group5/pullRequest?prid=9414198)
```suggestion
    if (document.getElementById('add-task-form').classList.contains("hidden"))
```